Italian/main/Settings.apk/res/values-it/strings.xml

```diff
@@ -1238,8 +1238,8 @@ Considera l'utilizzo di un ^1 più veloce per prestazioni migliori."</string>
     <string name="error_apn_empty">Il campo APN non può essere vuoto.</string>
     <string name="error_mcc_not3">Il campo MCC deve contenere 3 cifre.</string>
     <string name="error_mnc_not23">Il campo MNC deve contenere 2 o 3 cifre.</string>
-    <string name="restore_default_apn">Ripristino delle impostazioni APN predefinite.</string>
-    <string name="menu_restore">Ripristina imp. predefinite</string>
+    <string name="restore_default_apn">Ripristino delle impostazioni APN predefinite…</string>
```

Comment:
il messaggio appare temporaneamente cliccando il bottone che corrisponde alla riga 1242. in pratica è un messaggio del tipo "Operazione in corso..."